packages/kernel-modules/kernel-modules/finalize.yaml

```diff
@@ -1,2 +1,3 @@
 install:
-- depmod -a
\ No newline at end of file
+- depmod -a
```

Comment:
https://github.com/geaaru/luet-specs/blob/master/packages/kernels/mocaccino-lts/finalize.yaml#L2
It's better to specify the kernel version to elaborate because by default it uses the running kernel.